app/views/director/list.scala.html

```diff
@@ -0,0 +1,162 @@
+                            '_class -> "ui search button",
+                            '_label -> ""
+                        )
+                    </td>
+                    <td>
+                        @inputText(
+                            filterForm("heightMin"),
+                            '_label -> "",
+                            'type -> "number"
+                        )
+                        @inputText(
+                            filterForm("heightMax"),
+                            '_label -> "",
+                            'type -> "number"
+                        )
+                    </td>
+                    <td>
+                        @select(
+                            filterForm("gender"),
+                            Gender.values.toSeq.sortBy(_.actualName).map{genderVal => (genderVal.dbName, genderVal.actualName)},
+                            '_class -> "ui search button",
+                            '_label -> ""
+                        )
+                    </td>
+                    <td>
+                        <button
+                                style="border: none; background-color: transparent; padding: 0; cursor: pointer"
+                                type="button"
+                                onclick="location.href='@modules.director.routes.DirectorController.list()'">
+                            <i class="undo icon"></i>
+                        </button>
+                    </td>
+                </tr>
+                @Option(directors.items).filter(_.nonEmpty).map { directors =>
+                    @directors.map { director =>
+                        <tr>
+                            <td>
+                                @director.firstName
+                                @director.lastName
+                            </td>
+                            <td>@director.birthDate</td>
+                            <td>@director.nationality.nationality</td>
+                            <td>@director.height</td>
+                            <td>@director.gender.actualName</td>
+                            <td style="width: 70px">
+                                <button
+                                        style="border: none; background-color: transparent; padding: 0; cursor: pointer"
+                                        type="button"
+                                        onclick="location.href='@modules.director.routes.DirectorController.edit(director.id)'">
+                                    <i class="edit outline icon"></i>
+                                </button>
+                                <button
+                                        style="border: none; background-color: transparent; padding: 0; cursor: pointer"
+                                        type="button"
+                                        onclick="deleteRow('@modules.director.routes.DirectorController.delete(director.id)', '/directors')"
```

Comment:
Why use onclick Javascript event instead of simple `<a>` tag styled as a button? Semantic UI provides solution for styling.

Reply:
I thought links are only for GETs